libyul/AsmParser.cpp

```diff
@@ -393,21 +321,8 @@ Parser::ElementaryOperation Parser::parseElementaryOperation()
 		{
```

Comment:
Comment above has to be updated, we are not searching instructions anymore.

Reply:
Not just this. I think the comment can be removed completely, since it's a trivial `if A else B` case now.